MCprep_addon/spawner/entities.py

```diff
@@ -127,10 +129,10 @@ def updateEntityList(context):
 
 class face_struct():
 	"""Structure class for preprocessed faces of a mesh"""
-	def __init__(self, normal_coord, global_coord, local_coord):
-		self.n = normal_coord
-		self.g = global_coord
-		self.l = local_coord
+	def __init__(self, normal_coord: VectorType, global_coord: VectorType, local_coord: VectorType):
+		self.n: VectorType = normal_coord
+		self.g: VectorType = global_coord
+		self.l: VectorType = local_coord
```

Comment:
This should be a `dataclass` now that we can use them
```py
@dataclass
class face_struct():
	n: VectorType
	g: VectorType
	l: VectorType
```

I would also personally make the class be defined as the following to follow naming convention:
```py
@dataclass
class FaceStruct:
	normal_coord: VectorType
	global_coord: VectorType
	local_coord: VectorType
```